packages/logger/src/Logger.ts

```diff
@@ -1378,6 +1378,14 @@ class Logger extends Utility implements LoggerInterface {
       this.#bufferConfig.bufferAtVerbosity =
         LogLevelThreshold[bufferAtLogLevel];
     }
+    const AlcLogLevel =
+      this.getEnvVarsService().getAwsLogLevel() as keyof typeof LogLevelThreshold;
+
+    if (LogLevelThreshold[AlcLogLevel] > this.#bufferConfig.bufferAtVerbosity) {
+      this.#warnOnce(
+        'Advanced Loggging Controls (ALC) Log Level is higher than Log Buffering Log Level. Buffered logs will be filtered by ALC'
+      );
```

Comment:
This is quite a verbose message. There might be a better way to phrase it

Reply:
I don't mind the verbosity of the message, but I would reword the "is higher" to "is less verbose" to make it extra clear of what's happening.

I'm pretty sure if we ask 2 people which one is higher/lower in the context of log verbosity they'll give you two different answers.

Reply:
Agreed. I got confused whilst making the PR